examples/permissioned-voting/assets/asa.yaml

```diff
@@ -0,0 +1,14 @@
+vote-token:
+    total: 5912599999515
+    decimals: 0
+    defaultFrozen: false
+    unitName: "VOTETK"
+    url: "url"
+    # User may get "signature validation failed" from node if shorter hash is used.
+    metadataHash: "12312442142141241244444411111133"
+    noteb64: "noteb64"
+    manager: "WWYNX3TKQYVEREVSW6QQP3SXSFOCE3SKUSEIVJ7YAGUPEACNI5UGI4DZCE"
+    reserve: "WWYNX3TKQYVEREVSW6QQP3SXSFOCE3SKUSEIVJ7YAGUPEACNI5UGI4DZCE"
+    freeze: "WWYNX3TKQYVEREVSW6QQP3SXSFOCE3SKUSEIVJ7YAGUPEACNI5UGI4DZCE"
+    clawback: "WWYNX3TKQYVEREVSW6QQP3SXSFOCE3SKUSEIVJ7YAGUPEACNI5UGI4DZCE"
```

Comment:
let's remove `clawback` - we don't need that someone will transfer our vote rights. It has to be removed from here. Also, we please document in the README that we deliberately set it to empty.

Reply:
ok